dev-libs/libmspack/libmspack-0.10.1_alpha-r1.ebuild

```
@@ -21,53 +21,48 @@ HOMEPAGE="https://www.cabextract.org.uk/libmspack/"
 
 LICENSE="LGPL-2.1"
 SLOT="0"
-IUSE="debug doc static-libs"
-
-DEPEND=""
-RDEPEND=""
+IUSE="debug doc"
 
 S="${WORKDIR}/${MY_P}"
 
 src_prepare() {
-	if [[ ${PV} == "9999" ]] ; then
+	if [[ ${PV} == 9999 ]] ; then
```

```suggestion
	if [[ ${PV} == *9999 ]] ; then
```
and change the one below the inherit line too